pages/index.js

```diff
@@ -141,5 +144,16 @@ export default function Page() {
   )
 }
 
+export async function getStaticProps() {
```

Comment:
this is overcomplicating a piece of code that should require zero brain power to read.

Do it like this:
![image](https://user-images.githubusercontent.com/46534486/153958242-d90df951-e8c4-4b32-9502-2606dbbdf600.png)

Reply:
fixed

Reply:
No it's not. We can talk about this on Wednesday.

Reply:
oh i get what you mean